app/members/views.py

```diff
@@ -3,8 +3,11 @@
 from django.contrib.auth.decorators import login_required
 
 def index(request):
+	#
     return HttpResponse(loader.get_template('index.html').render(Context({})))
 
 @login_required(login_url='/')
 def authenticated_page(request):
-    return HttpResponse(loader.get_template('auth.html').render(Context({})))
+	#
+	context = RequestContext(request, {})
+	return HttpResponse(loader.get_template('auth.html').render(context))
```

Comment:
Looks like you have TAB indenting here, which is not recommended in python as it gets very confusing and can cause issues, the standard generally is to use spaces which is an option in sublime text